### PR TITLE
upgrade vault-plugin-auth-azure to v0.13.0

### PR DESCRIPTION
**Pull request by vinay-gopalan**
Upgrades the Auth Azure plugin to `v0.13.0`

Steps:
```
go get github.com/hashicorp/vault-plugin-auth-azure@v0.13.0
go mod tidy
```

### CODE REVIEW

File: changelog/19077.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+```release-note:feature
+auth/azure: Adds support for rotate root to Azure Auth
```
Comment (vinay-gopalan) [inline changelog/19077.txt:2]: Wasn't sure if I should list all the other features added or just the latest one. Thoughts?

Comment (robmonte) [reply changelog/19077.txt:2]: I think if they are all being added to 1.13 and there's no previous changelog entry for them, it makes sense to add them all now.

**Review by robmonte — APPROVED** [review]
👍

File: changelog/19077.txt (hunk 1 above)
Comment (austingebauer) [reply changelog/19077.txt:2]: I'd say list all of the features/improvements/fixes that are being brought in at this time. I'd say you can leave the dependency upgrade entry out of the Vault changelog though.

Comment (austingebauer) [reply changelog/19077.txt:2]: Also, there is a specific format for "feature" changelog entries. I don't believe these conform to the format.

Comment (vinay-gopalan) [reply changelog/19077.txt]: Ah thanks for the clarification, @austingebauer! Updated the changelog to match the desired format for multiple features

**Review by Zlaticanin — APPROVED** [review]
👍

**Pull request merged**